CMakeLists.txt

```diff
@@ -16,13 +16,25 @@ include(FairMQLib)
 get_git_version()
 
 project(FairMQ VERSION ${PROJECT_VERSION} LANGUAGES CXX)
-message(STATUS "${BWhite}${PROJECT_NAME}${CR} ${PROJECT_GIT_VERSION} from ${PROJECT_DATE}")
 
-set(PROJECT_MIN_CXX_STANDARD 17)
+include(CTest)
+include(BundlePackages)
+
+set(PROJECT_FairCMakeModules_VERSION 0.2)
+find_package(FairCMakeModules ${PROJECT_FairCMakeModules_VERSION} QUIET)
+if(NOT FairCMakeModules_FOUND)
+  build_bundled(FairCMakeModules extern/FairCMakeModules)
+  find_package(FairCMakeModules REQUIRED)
+endif()
+
+include(FairFindPackage2)
+include(FairFormattedOutput)
+include(FairSummary)
```

Comment:
This is included twice? here and

Reply:
Thx, I added another commit which refactors the whole CMake into topic-based modules and adds include_guards, so each module should now `include()` everything it directly depends on. But this particular include here was just redundant, it is gone.